### PR TITLE
Added Jekyll and converted index.html to Liquid template

### PR DESCRIPTION
**Pull request by lili2311**
## What are the changes included in this PR?
- Added Jekyll dependency
- Updated README instructions to cover Jekyll required dependencies
- Converted index.html to use Liquid templates, this can be used as an example to convert the rest
- Amended the build and deploy in the Gruntfile to reflect Jekyll changes.
- Split footer, nav and social sections into separate html includes.
## If the changes are visual, provide a screenshot.

no visual changes as the homepage renders as usual

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,15 +12,17 @@
     "grunt": "^0.4.5",
     "grunt-aws-s3": "~0.14.3",
     "grunt-browser-sync": "~2.2.0",
+    "grunt-concurrent": "^2.3.0",
```
Comment (tanyapowell) [inline package.json:15]: what is this dependency for?

File: README.md [hunk 2]
```diff
@@ -30,6 +34,24 @@ Women Hack for Non-Profits website rebuild.
 	"bucket": "www.womenhackfornonprofits.com",
 	"region": "eu-west-1"
 }
+```
+
+## Development Instructions
+
+1. HTML changes 
+	In order for Jekyll to watch and compile all the changes to html run `ekyll serve` in the `src` directory
```
Comment (tanyapowell) [inline README.md]: missing the `j` in `jekyll`? 😄

File: src/404.html [hunk 3]
```diff
@@ -115,7 +114,7 @@ <h2 class="section-heading">Newsletter</h2>
       </ul>
     </nav>
     <footer>
```
Comment (tanyapowell) [inline src/404.html:116]: Shouldn't this footer tag now be removed and replaced with `{% include footer.html %}`?

File: src/_includes/social.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<nav class="social">
```
Comment (tanyapowell) [inline src/_includes/social.html]: can this file be included in the `footer.html` so there's less repetition of the same code throughout the codebase?

**Comment by tanyapowell** [comment]
I'm a bit confused by the `footer`...
At the bottom of each page there is current

File: src/404.html (hunk 3 above)
Comment (lili2311) [inline src/404.html:116]: I only converted 1 file index.html I can do all but thought i might leave some for someone else to do :D Currently you can convert one by one without affecting the build/deploy

File: package.json (hunk 1 above)
Comment (lili2311) [inline package.json:15]: to allow to run jekyll, sass and watch at once!

**Comment by lili2311** [comment]
Fixed jekyll typo

**Comment by tanyapowell** [comment]
👍 after the conflicts have been resolved 😄

**Pull request merged**